LPPC/Plugins/topsky/GRpluginSettings.txt

```diff
@@ -118,6 +118,8 @@ Airport_Elevation=374
 Airport_Radius=6.5
 Airport_Runway_End=21:N038.47.50.400:W009.07.38.500
 Airport_Runway_End=03:N038.45.56.460:W009.08.39.480
+Airport_Runway_End=35:N008.47.50.400:W089.07.38.500
+Airport_Runway_End=17:N008.45.56.460:W089.08.39.480
```

Comment:
tricked the system to no longer trigger runway related warnings by temporarily dislodging "rwy 35" to the pacific ocean off the coast of costa rica, enjoy your trip to central america